types/parcel-bundler/index.d.ts

```diff
@@ -173,6 +173,8 @@ declare class ParcelBundler {
     addPackager(type: string, packager: string): void;
 
     bundle(): Promise<ParcelBundler.ParcelBundle>;
+
+    middleware(): (req: any, res: any, next: any) => any;
```

Comment:
Maybe you should use [`Request`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0d4dfe2e4642dccb8d078342f55bf125477f0b57/types/request/index.d.ts#L216) and [`Response`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0d4dfe2e4642dccb8d078342f55bf125477f0b57/types/request/index.d.ts#L311)

But i'm not sure

I am not sure about the `next`'s type

```typescript
middleware(): (req: Request, res: Response, next?: () => any) => void;
```

Reply:
OK, I added additional dependency express-serve-static-core and correct parameter types.